backend/typescript/migrations/2024.07.27T17.21.42.add-status-to-user-table.ts

```diff
@@ -0,0 +1,14 @@
+import { DataType } from "sequelize-typescript";
+import { Migration } from "../umzug";
+
+export const up: Migration = async ({ context: sequelize }) => {
+  await sequelize.getQueryInterface().addColumn("users", "status", {
+    type: DataType.ENUM("Active", "Inactive"),
```

Comment:
i'm okay with `Inactive` but i'm wondering if `Invited` would be a better term? to me `Inactive` is more like they no longer use this account, perhaps we can do `Active`, `Inactive`, `Invited` ?? this might be a question for elle LOL

Reply:
done